frontend/assets/style/components/_forms.scss

```diff
@@ -129,17 +130,23 @@ textarea,
     border-right: 1px solid $general_0;
   }
 
-  label {
+  .sufix {
```

Comment:
FYI, the correct English spelling is `.suffix` (if that matters)

Reply:
yes, `.suffix.` fixed :D